graphbolt/src/cuda/common.h

```diff
@@ -53,7 +59,7 @@ struct CUDAWorkspaceAllocator {
   // Required by thrust to satisfy allocator requirements.
   value_type* allocate(std::ptrdiff_t size) const {
     return reinterpret_cast<value_type*>(
-        c10::cuda::CUDACachingAllocator::raw_alloc(size));
+        c10::cuda::CUDACachingAllocator::raw_alloc(size * sizeof(value_type)));
```

Comment:
Do we need a callback to handler the allocating failure cases?

Reply:
This change is not about the logic, it is modifying the existing class so that it can be reused as an allocator for `cuco::static_map`.